dataflow/src/test/java/busyexpr/BusyExpression.java

```diff
@@ -0,0 +1,30 @@
+package busyexpr;
```

Comment:
Use the same package name for the checker and its tests, I don't have a strong preference whether it's the short or long name. Which do you like better?
(The classes use e.g. `BusyExprStore`, so maybe there is more use of the shorter `busyexpr`.)

Reply:
Thanks for noticing this!